File: app/src/main/java/com/example/myappcontacts/presentation/contactsmap/presenter/IContactsMapPresenter.java
package com.example.myappcontacts.presentation.contactsmap.presenter;

import com.google.android.gms.maps.model.Marker;

import java.util.Map;
import java.util.UUID;

public interface IContactsMapPresenter {

    void init();

    void setShownMarkersMap(Map<Marker, UUID> markersMap);

    void onMarkerClicked(Marker marker);
}

File: app/src/main/java/com/example/myappcontacts/presentation/contactsmap/presenter/IContactsMapPresenter.kt
package com.example.myappcontacts.presentation.contactsmap.presenter

import com.google.android.gms.maps.model.Marker
import java.util.UUID

interface IContactsMapPresenter {

    fun init()

    fun setShownMarkersMap(markersMap: Map<Marker, UUID>)

    fun onMarkerClicked(marker: Marker)
}
